DESCRIPTION

```diff
@@ -1,4 +1,18 @@
 Title: Explorer designed to visualize complex multi-year multi-specimen data. Case study of Slovenian wolves.
+Imports: 
```

Comment:
Sadly, shiny doesn't support the 'package style' imports, suggests and the like. So far the only mechanism that programmatically handles this is `rsconnect` package which is used to e.g. deploy to shinyapps.io. It scrapes the files and searches for `require` and `library` statements.

That being said, having this in here doesn't really hurt. @zkuralt please also add versions (use whatever is working for you right now).